bin/app.R

```diff
@@ -1127,7 +1127,7 @@ server <- function(input, output, session) {
         message <- paste("Calculating space use...")
         showNotification(message, id = mid, type = "message", duration = NULL,
                          session = session)
-        results <- createContour(raster, probs, input$zone, input$datum, input$mkde_buffer)
```

Comment:
@matthewwangg ,oddly the PR shows the input$mkde_buffer when both the [refactordev](https://github.com/sinkovit/Spatial-ecology/blob/3b025a277055e2d87d6e11e0947ff79ee4383d22/bin/app.R#L1130) and [dev](https://github.com/sinkovit/Spatial-ecology/blob/621d2b68df03cee25447d8c6a28ce4d0d27b1f28/bin/app.R#L1131) branches don't have that anymore...

Reply:
That was changed 2 weeks ago so maybe we fixed it on both branches already, not sure why it shows up but it can't hurt to include since the new name is the correct one right